Kinvey/Kinvey/Client.swift

```diff
@@ -267,6 +273,7 @@ open class Client: Credential {
         if let user = keychain.user {
             user.client = self
             activeUser = user
+            clientId = keychain.clientId
             let customUser = user as! U
             completionHandler(.success(customUser))
         } else if let kinveyAuth = sharedKeychain?.kinveyAuth {
```

Comment:
I don't completely understand your question. `client_id` should be stored regardless of SSO (whether the app is using SSO or not). If you're thinking about how the `client_id` factors into multiple apps that use SSO, it's a good question to ask @andrewmaillet